### PR TITLE
release v3.2.1

### PR DESCRIPTION
**Pull request by flavorjones**
changelog:

### Added

- Improve `pp` output for `ActiveHash::Relation`. [#288](https://github.com/active-hash/active_hash/pull/288) @flavorjones

### Fixed

- Fix relation matching when attribute name collides with a method. [#281](https://github.com/active-hash/active_hash/pull/281) @flavorjones
- Fix association reflection in applications that don't use ActiveHash::Associations. [#286](https://github.com/active-hash/active_hash/pull/286) @iberianpig
- Fix `ActiveHash::Relation#method_missing` and `#respond_to_missing?` without scopes. [#278](https://github.com/active-hash/active_hash/pull/278) @julianrubisch

### CODE REVIEW

**Comment by flavorjones** [comment]
@kbrock Whenever you approve I'll merge and ship the gem release

**Comment by flavorjones** [comment]
@kbrock When you get a chance, this PR needs an approval before I can merge it.

**Comment by flavorjones** [comment]
@kbrock When you get a chance, this PR needs an approval before I can merge it.

**Review by kbrock — APPROVED** [review]

**Comment by kbrock** [comment]
thnx for the repost. not sure how this slipped by

**Comment by flavorjones** [comment]
Pushed gem and created a Release: https://github.com/active-hash/active_hash/releases/tag/v3.2.1

Thank you @kbrock!

**Pull request merged**
